packaging/installer/methods/packages.md

```diff
@@ -2,46 +2,78 @@
 
-We provide two separate repositories, one for our stable releases and one for our nightly releases.
+-   Nightly releases: [netdata/netdata-edge](https://packagecloud.io/netdata/netdata-edge)
+-   Stable releases: [netdata/netdata](https://packagecloud.io/netdata/netdata)
 
-1.  Stable releases: Our stable production releases are hosted in
-    [netdata/netdata](https://packagecloud.io/netdata/netdata) repository of package cloud
-2.  Nightly releases: Our latest releases are hosted in
-    [netdata/netdata-edge](https://packagecloud.io/netdata/netdata-edge) repository of package cloud
+Read our notice about [nightly vs. stable releases](../README.md#nightly-vs-stable-releases) to understand the
+differences between the two.
 
-Visit the repository pages and follow the quick set-up instructions to get started.
+## Quickstart
 
-## Using caching proxies with PackageCloud repositories
+packagecloud offers helper installation scripts to add their repositories to your package manager. Jump to your
+distribution and use the commands to install Netdata get _automatic nightly updates_ via your distribution's package
+manager. You can also manually add the repositories on [`.deb`
+systems](https://packagecloud.io/netdata/netdata-edge/install#manual-deb) or [`.rpm`
+systems](https://packagecloud.io/netdata/netdata-edge/install#manual-rpm).
 
-PackageCloud only provides HTTPS access to repositories they host, which
-means in turn that Netdata's package repositories are only accessible
-via HTTPS. This is known to cause issues with some setups that use a
-caching proxy for package downloads.
+-   [Ubuntu and Debian (`.deb`)](#ubuntu-and-debian-deb)
+-   [Fedora (`.rpm`)](#fedora-rpm)
+
+### Ubuntu and Debian (`.deb`)
+
+Use the helper script to add the packagecloud repository, then use `apt` to install Netdata.
+
+```bash
+curl -s https://packagecloud.io/install/repositories/netdata/netdata-edge/script.deb.sh | sudo bash
+sudo apt-get update
+sudo apt-get install netdata
+```
+
+Skip ahead to the [What's next?](#whats-next) section to find links to helpful post-installation guides.
+
+### Fedora (`.rpm`)
+
+Use the helper script to add the packagecloud repository, then use `dnf` to install Netdata.
+
+For `.rpm` systems using YUM (Fedora, CentOS, RHEL, OpenSuSE):
+
+```bash
+curl -s https://packagecloud.io/install/repositories/netdata/netdata-edge/script.rpm.sh | sudo bash
+sudo dnf update
+sudo dnf install netdata
+```
+
+Skip ahead to the [What's next?](#whats-next) section to find links to helpful post-installation guides.
+
+### Other operating systems
+
+While RHEL, CentOS, and Open SuSE all support `.rpm` packages, full support via these packages is still a
+work-in-progress. See our [distribution matrix](../../DISTRIBUTIONS.md) to understand whether you can install Netdata
+via these `.rpm` packages, and then use your package manager to add the packagecloud repository and install Netdata.
```

Comment:
Do we have confirmation that SuSE doesn't work? I mean, Tumbleweed doesn't because we provide no packages for it, and SLES probably doesn't, but I'm fairly certain based on manual testing that Leap works just fine for recent versions (and if it doesn't, we should have a bug report for that.